### ISSUE TITLE
Accessibility - Regions

### ISSUE THREAD

**Issue by rohitpaul** [CMSgov/eAPD#2502]
Proper regions should be assigned to the different parts of the web page.

### This task is done when...
 - [ ] Top should be labeled with region "banner" or "header".
 - [x] Left side navigation should be labeled with region "nav".
 - [x] Main content should be labeled with region "main".

Reference: [w3.org](https://www.w3.org/WAI/tutorials/page-structure/regions/)

**Comment by tbolt** [comment]
The heading sections were already using <header>
